include/psa/crypto_values.h

```diff
@@ -1571,6 +1601,22 @@
  */
 #define PSA_KEY_ID_VENDOR_MAX                   ((psa_app_key_id_t)0x7fffffff)
 
+/** A key wrapping key suitable for keys managed directly by the
+ * PSA Cryptography implementation.
+ *
+ * Applications may use this key with psa_wrap_key_with_policy() and
+ * psa_unwrap_key_with_policy() to back up a key that has the
+ * #PSA_KEY_USAGE_BACKUP usage flag and later restore it on the same device,
+ * if the implementation supports it.
+ *
+ * The existence of this key is optional. Implementations that support
+ * key wrapping should offer a key with this identifier if having a
+ * single wrapping key makes sense in the scenarios where this implementation
+ * is used. If this key exists, it must be private to the device, i.e. it
+ * must not be shared with any other device or entity.
+ */
+#define PSA_KEY_ID_WRAP_BOUND                   ((psa_app_key_id_t)0x80000062)
```

Comment:
Is there any particular significance to the value chosen? - why MIN_UINT32 + 0x62?

Reply:
Not enamoured with the name, as we haven't used "bound" anywhere else in the API or documentation. Is `PSA_KEY_ID_WRAP_BUILT_IN` any better, or worse?

Reply:
0x80000000 is the start of the range reserved for ids defined by PSA.

`0x80000000 + 'b'` (for “bound”) because why not.

I'm very open to naming suggestions. Once a few people have made theirs I'll do a pass of renaming stuff with an eye to consistency and the other eye to comprehensibility.